api/routes.go

```diff
@@ -125,6 +124,34 @@ func (App *API) RoutesDeleteHandler(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (App *API) RoutesEditHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
This needs to have a giant comment saying it only allows editing of a route's `enabled` flag.